istioctl/cmd/istioctl/main.go

```diff
@@ -33,8 +37,28 @@ func main() {
 
 	rootCmd := cmd.GetRootCmd(os.Args[1:])
 
+	EnableKlogWithCobra()
+
 	if err := rootCmd.Execute(); err != nil {
 		exitCode := cmd.GetExitCode(err)
 		os.Exit(exitCode)
 	}
 }
+
+// EnableKlogWithCobra enables klog to work with cobra / pflags.
+// k8s libraries like client-go use klog.
+func EnableKlogWithCobra() {
+	klog.InitFlags(nil)
```

Comment:
does passing nil (which then uses goflag.CommandLine) cause any issues here in help messages, etc? Should we create a dedicated FlagSet for this so it doesn't unexpectedly leak out?